torchvision/transforms/transforms.py

```diff
@@ -2104,8 +2104,12 @@ def __init__(self, alpha=50.0, sigma=5.0, interpolation=InterpolationMode.BILINE
             interpolation = _interpolation_modes_from_int(interpolation)
         self.interpolation = interpolation
 
-        if not isinstance(fill, (int, float)):
-            raise TypeError(f"fill should be int or float. Got {type(fill)}")
+        if isinstance(fill, (int, float)):
+            fill = [float(fill)]
+        elif isinstance(fill, (list, tuple)):
+            fill = [float(f) for f in fill]
```

Comment:
Do we actually need to convert to float?

Reply:
Unfortunately, we do due to JIT :smiling_face_with_tear: Removing the float conversion from L2110 gives us

```python
import torch.jit

from torchvision import transforms

torch.jit.script(transforms.ElasticTransform(fill=[1]))
```

```
[...]
Expected a value of type 'Optional[List[float]]' for argument 'fill' but instead found type 'List[int]'.
[...]
```

I know this is ugly AF and so far from being Pythonic, but given that it is on v1 I really don't want to deal with this any more than I have to.